### ISSUE TITLE
very low audio level

### ISSUE THREAD

**Issue by jbbenoit73** [amiaopensource/vrecord#716]
Hello,
The recording audio level is really low compared to the input level. What can i do ?
Thanks,
jb

**Comment by privatezero** [comment]
Hi @jbbenoit73, to clarify - are you using a Blackmagic device and is this during audio only capture or video/audio capture? If using a Blackmagic device, is your audio coming in via the analog inputs or the SDI input?

**Comment by jbbenoit73** [comment]
Hello ! Thanks for answer. I did various tests with a dv test tape calibrated at 1000 hz at different level | -18 db | -12db | -6db | -3db | 0db. SDI video input (DV-> S-video output -> Converter SDI input -> BMDecklink SDI; DV -> analog audio output -> analog BMDecklink inputs. I had a difference of 10 db constant with Vrecord. Then i tested with MediaExpress, the difference was the same. I changed the parameters in BM Desktop Setup. By selecting 'Use Hifi level'. The difference drops to +/- 3 db using either MediaExpress or Vrecord. It's OK.
Best regards.
jb
